cloudformation/database/serverless-database.yaml

```diff
@@ -145,8 +145,8 @@ Resources:
       Tags:
         - Key: Name
           Value: ServerlessDatabaseClusterSubnet1
-        - Key: 'parallel-cluster:accounting'
-          Value: !Sub cluster-subnet-${AWS::Region}a
+        - Key: 'parallelcluster:infrastructure'
+          Value: 'accounting'
```

Comment:
Minor (here and elsewhere): I'd say more specifically "slurm accounting" rather than "accounting"

Reply:
Changed.